### PR TITLE
upgrade deps to fix m1 builds

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

Upgrade deps to fix m1 builds

[Related doc PR](https://github.com/mattermost/mattermost-developer-documentation/pull/860)

[Context](https://community-daily.mattermost.com/core/pl/di7a1yhkjf8dtp31e15rb8t61h)

#### Ticket Link

No ticket

#### Device Information

This PR was tested on: mac m1 ios simulator

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: ios/Gemfile [hunk 1]
```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gem "cocoapods", "1.10.1"
+gem "ethon", ">=0.13.0"
```
Comment (enahum) [inline ios/Gemfile:4]: why not use a specific version?

File: ios/Gemfile.lock [hunk 2]
```diff
@@ -70,24 +70,27 @@ GEM
...
 PLATFORMS
   ruby
 
 DEPENDENCIES
   cocoapods (= 1.10.1)
+  ethon (>= 0.13.0)
 
 BUNDLED WITH
-   2.1.4
+   2.2.22
```
Comment (enahum) [inline ios/Gemfile.lock:96]: in the docs you talk about using bundler 2.1.4 which was previously used but here I see bundler 2.2.22, is that intended and should we bump the version everywhere ?

**Review by enahum — CHANGES_REQUESTED** [review]

File: ios/Gemfile.lock (hunk 2 above)
Comment (Willyfrog) [reply ios/Gemfile.lock:96]: this was probably bumped when asked to upgrade some other files, 0/5

File: ios/Gemfile (hunk 1 above)
Comment (Willyfrog) [reply ios/Gemfile:4]: yes, we can probably fix this one

**Comment by mkraft** [comment]
I've pulled this PR and am still unable to successfully execute `bundle exec pod install` in the ios directory.

**Comment by Willyfrog** [comment]
Hi @mkraft which problem are you running into? Maybe I overlooked an step?

**Comment by mkraft** [comment]
> Hi @mkraft which problem are you running into? Maybe I overlooked an step?

I was getting stuck on the `pod install` step, but this works for me:

```shell
bundle exec arch -x86_64 pod install
```

**Comment by Willyfrog** [comment]
I don't recall having to do that, but I'll keep it in mind in case I see the issue, as we might need to update the docs if that's the case, thanks for the tip

**Comment by enahum** [comment]
This PR should not be needed anymore

**Pull request closed without merging**